docs/specifications/1-introduction.md

```diff
@@ -0,0 +1,172 @@
+- Relevant additions or modifications to the compliance suite
+- Implementation of the new behavior, including new or updated tests in all the
+  language bindings
+
+The pull request's body must reference the RFC if there has been one, and
+otherwise must include all discussion necessary to explain the reasoning behind
+the proposal (including alternatives considered, risks, ...).
+
+#### <a id="addressing-gaps"/> :mag: Addressing Gaps
+
+Proposal that increase the specification's coverage (desribing behavior that
+already exists) are handled as GitHub pull request that must contain the
+following elements:
+
+- Relevant additions to the specification documents
+- New compliance test(s) that enshrine the described behavior
+- Implementation of the new compliance test(s) for all *Generally Available*
+  language bindings
+
+The pull request body should provide pointers to any and all elements that can
+be used to verify that the behavior that is described is indeed what is
+currently implemented.
+
+#### <a id="trivial"/> :thumbsup: Trivial Changes
+
+Proposal of trivial changes, such as correcting typos in the document, or
+re-phrasing elements of the specification without altering the meaning
+(typically to improve clarity) are handled in a simple GitHub pull request.
+
+
+## Design Tenets (unless you know better ones)
+
+* *jsii* APIs strive to feel idiomatic in all supported languages.
+* *jsii* applications behave identically regardless of the language they are
+  written in. It favors correctness over performance.
+* *jsii* **does not** attempt to support all TypeScript idioms (many features of
+  TypeScript cannot be expressed in some target languages).
+  * When prohibiting an idiom, *jsii* strives to provide an error message that
+    gives the user insight into why the pattern cannot be supported.
+* *jsii* does not force API design opinions on the developer:
+  * Reserved names are limited to a minimum.
+  * TypeScript API design patterns that will not transpose well in other
+    languages may cause a warning to be issued, but the developer is ultimately
+    entitled to decide whether they want to take or leave the advice.
+* *jsii* uses idiomatic tools whenever possible:
+  * Generated libraries can be easily published to the "standard" package
+    repository for the language.
+  * Standard tools can be used to work with the generated libraries, and do not
+    require any special configuration.
+
+
+## Concepts
+
+*jsii* allows developers to author code once in **TypeScript**, while allowing
+use in a variety of other programming languages (including **C#**, **Java** and
+**Python**).
+
+### Assemblies
+
+The *jsii Assembly* document contains a specific representation of the API
+exported by the **TypeScript** module. Similar to a header file in the **C++**
+world, it contains only information about the signatures of APIs (type names
+with method and property signatures, ...) and no implementation.
```

Comment:
(jsii noob here) does it also not contain documentation for the methods and types?